src/OrchardCore/OrchardCore.Data.YesSql/DbConnectionValidator.cs

```diff
@@ -145,7 +148,7 @@ private ISqlBuilder GetSelectBuilderForDocumentTable(string tablePrefix, string
 
         selectBuilder.Select();
         selectBuilder.Selector("*");
-        selectBuilder.Table(_yesSqlOptions.TableNameConvention.GetDocumentTable());
+        selectBuilder.Table(_yesSqlOptions.TableNameConvention.GetDocumentTable(), alias: null, _yesSqlConfiguration.Schema);
```

Comment:
Remove ` IOptions<YesSqlOptions> yesSqlOptions,` from being injected and use the following instead

```suggestion
        selectBuilder.Table(_yesSqlConfiguration.TableNameConvention.GetDocumentTable(), alias: null, _yesSqlConfiguration.Schema);
```

Reply:
I think we still need `YesSqlOptions` for table prefix separator

Reply:
yeah true. but I would read the `TableNameConvention` from the `IConfiguration` instead